java/src/jmri/jmrit/display/layoutEditor/LayoutEditor.java

```diff
@@ -289,6 +251,115 @@
     // grid size in pixels
     private int gridSize = 10;
 
+
+    // size of point boxes
+    @Deprecated
+    public static final double SIZE = 3.0;
```

Comment:
SIZE and SIZE2 do not need to be deprecated or made public. private methods and fields are private; they can be changed at will without issue, its only the public methods and fields that need to go through the deprecation before removal process.